xbrowse/core/constants.py

```diff
@@ -394,12 +420,16 @@
     'name': 'intergenic_variant',
     'slug': 'intergenic_variant',
     'so': 'SO:0001628'},
-
 ]
 
+ANNOTATION_DEFINITIONS = CLINVAR_ANNOTATION_DEFINITIONS + VEP_ANNOTATION_DEFINITIONS
+ANNOTATION_DEFINITIONS_INTERNAL = ANNOTATION_DEFINITIONS + HGMD_ANNOTATION_DEFINITIONS + VEP_ANNOTATION_DEFINITIONS
+
```

Comment:
`ANNOTATION_DEFINITIONS` already  includes `VEP_ANNOTATION_DEFINITIONS` so you are actually adding it twice to `ANNOTATION_DEFINITIONS_INTERNAL`

Reply:
oops.. good call